payment-management-service/payment-management-server/src/main/java/agh/fis/payment_management/service/PaymentService.java

```diff
@@ -67,36 +67,37 @@ public PaymentDto getPaymentById(Integer id){
         throw new ResponseStatusException(HttpStatus.NO_CONTENT,  "Payment with given id does not exist");
     }
 
-    public PaymentDto deletePaymentById(Integer id){
+    public Void deletePaymentById(Integer id){
         Optional<PaymentEntity> paymentEntity = paymentRepository.findById(id);
         if (paymentEntity.isPresent()){
             paymentRepository.delete(paymentEntity.get());
             logger.info("Deleted Payment with id: {}", paymentEntity.get().getId());
-            return getPaymentDtoFromPaymentEntity(paymentEntity.get());
         }

```

Comment:
I think **else** is necessary

Reply:
what should the **else** block contain? Atm there are 2 branches in this function:
1. if Payment exists -> delete it
2. (else) Payment doesn't exist -> return status 204 with comment

Reply:
I will also fetch the optional once in a next PR